src/playground/services/invoices/retrieve/endpoint.clj

```diff
@@ -20,9 +20,5 @@
         record (->> (logic/to-query id)
                     (h/format)
                     (jdbc/query db)
-                    (first))
-        ]
-    (if record
-      {:status 200
-       :body   record}
-      {:status 404})))
+                    (first))]
```

Comment:
now without an escape, there is no 404 for a wrong path. ( I can't see why there will be a wrong path as these are linked from the `retrieve-all` which are the actual entries in the db, but is this a faulty design?)

Reply:
And why did you remove the if/else?

It seemed fine.

The `if (= id 0)` from https://github.com/manandearth/playground/pull/3/files/d909ab8f5d88b587eca0024d324818bc964134e6#diff-8325b44f98d0bc32c6eb2de1a1533ab6 was the wrong one.

Reply:
because I moved the ring-response to the service, and realised that I need to rethink where the the `:status` will be assigned. I will look for a different way.

Reply:
👍

Handlers are right place to deal with "http stuff": status, content-type, redirects, etc